exercises/twelve-days/canonical-data.json

```diff
@@ -0,0 +1,121 @@
+{
+  "exercise": "twelve-days",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "verse",
+      "cases": [
+        {
+          "description": "first day a partridge in a pear tree",
+          "property": "recite",
+          "start verse": 1,
```

Comment:
We have now come up with an updated format, which requires the properties to be lowerCamelCase (see #983). Could you update the property to `startVerse`? And do the same for the other "start verse" properties?